web/modules/custom/joinup_migrate/README.md

```diff
@@ -26,39 +26,55 @@ In order to start a migration we will need to do some preparation:
    previous paragraph.
 2. Put your local configuration into `build.properties.local`:
 
-    ```
-    # Migration configuration
-    # -----------------------
+```
+# Migration configuration
+# -----------------------
+
+# Database settings.
+migration.db.name = my_db_name
+migration.db.import_path = /my/path/to/d6-joinup.sql
+```
 
-    # Database settings.
-    migration.db.name = my_db_name
-    migration.db.user = my_db_user
-    migration.db.password = my_db_pass
-    migration.db.import_path = /my/path/to/d6-joinup.sql
-    ```
+Note that `migration.db.host`, `migration.db.port`, `migration.db.user` and
+`migration.db.password` are defaulting to the main database (D8) values as the
+migration is performing select queries that are joining tables across the two
+databases. For this reason both, source database and the destination database,
+should live on the same server and should be accessible by the same use. The
+MySQL user used to connect to the Drupal 8 site should be granted with read-only
+permissions against the D6 database, so he can read source data.
 
 3. Run the migration setup. Note that this should normally only be run once
    since it will write the migration database credentials to `settings.php`.
    Running this again will cause these credentials to be appended a second
    time which is useless.
 
-    ```
-    $ ./vendor/bin/phing setup-migration
-    ```
+```
```

Comment:
Maybe we can revert this. The reason I did not fix this here as well is that for an ordered list it will indeed indent the wrapper and will present it as a child of the ordered list item.
Lower in this readme, it was the same but outside the ordered list. This caused the wrapper to filter the ``` and present them as plain text. 
I think this feels better to be within the ordered list item rather than outside.